src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -477,15 +507,21 @@ def prepare_latents(
         generator,
         latents=None,
     ):
+        if latents is not None:
+            if latents.ndim == 4:
+                logger.warning(
+                    "Unpacked latents detected. These will be automatically packed. "
+                    "In the future, please provide packed latents to improve performance."
+                )
+                latents = self._pack_latents(latents, batch_size, num_channels_latents, height, width)
```

Comment:
Packing the latents is an inexpensive operation. So, I think this warning is unnecessary.

Reply:
Agreed, the warning can be removed.